integration-tests/alfresco-enterprise-74/overload.gradle

```diff
@@ -0,0 +1,8 @@
+ext {
+    alfrescoVersion = '7.4.1.3'
+    alfrescoVersionShort = '74'
+    alfrescoArtifactId = 'content-services'
+
+    mvcVersion = '8.0.0'
```

Comment:
Mayhaps we want to align with how ethias is configured.
(DE instead of MVC)

Although it should not matter...